src/prpy/util.py

```diff
@@ -725,6 +738,75 @@ def GeodesicDistance(t1, t2, r=1.0):
     error[3] = r*error[3]
     return numpy.linalg.norm(error)
 
+
+def GetGeodesicDistanceBetweenTransforms(T0, T1):
+    """
+    Wrapper, to match GetGeodesicDistanceBetweenQuaternions()
+    """
+
+    return GeodesicDistance(T0, T1, r=1.0)
```

Comment:
I'm not thrilled about hard-coding `r=1.0` here. This parameter has a huge effect on the behavior of the distance function.